### PR TITLE
HAWQ-1538. Install internal profiles definition file in conf directory.

### PR DESCRIPTION
**Pull request by sansanichfb**

### CODE REVIEW

File: pxf/build.gradle [hunk 1]
```diff
@@ -310,7 +309,7 @@ project('pxf-service') {
     }
 
     project.distTar {
-        from('src/main/resources/pxf-profiles-default.xml') { into 'conf' rename { 'pxf-profiles.xml' } }
```
Comment (shivzone) [inline pxf/build.gradle]: We might need both pxf-profile-default and pxf-profile in the tar

Comment (sansanichfb) [reply pxf/build.gradle]: Sure, we have them, here is layout:
{code}
office-5-181:gpdb adiachenko$ ls -alh pxf/conf
total 96
drwxr-xr-x  12 adiachenko  admin   408B Oct  9 17:07 .
drwxr-xr-x   8 adiachenko  admin   272B Oct  9 17:07 ..
-rw-r--r--   1 adiachenko  admin   1.3K Oct  9 17:07 pxf-env.sh
-rw-r--r--   1 adiachenko  admin   1.3K Oct  9 17:07 pxf-log4j.properties
-rw-r--r--   1 adiachenko  admin   3.1K Oct  9 17:07 pxf-private.classpath
-rw-r--r--   1 adiachenko  admin   2.7K Oct  9 17:07 pxf-privatebigtop.classpath
-rw-r--r--   1 adiachenko  admin   3.4K Oct  9 17:07 pxf-privatehdp.classpath
-rw-r--r--   1 adiachenko  admin   3.4K Oct  9 17:07 pxf-privatephd.classpath
-rw-r--r--   1 adiachenko  admin   9.3K Oct  9 17:07 pxf-profiles-default.xml
-rw-r--r--   1 adiachenko  admin   1.1K Oct  9 17:07 pxf-profiles.xml
-rw-r--r--   1 adiachenko  admin   1.2K Oct  9 17:07 pxf-public.classpath
-rw-r--r--   1 adiachenko  admin   1.3K Oct  9 17:07 pxf-site.xml
{code}

Comment (shivzone) [reply pxf/build.gradle]: Cool, as long as the above files exist with the right content we should be good

**Comment by lavjain** [comment]
If the fix is working please feel free to merge. It is hard to review this kind of change.

**Comment by sansanichfb** [comment]
@lavjain feel free to try it out by ```make install/tar/bundle DATABASE=gpdb```

**Pull request closed without merging**